actors/util/adt/intmap.go

```diff
@@ -0,0 +1,110 @@
+package adt
+
+import (
+	"bytes"
+
+	amt "github.com/filecoin-project/go-amt-ipld/v2"
+	cid "github.com/ipfs/go-cid"
+	errors "github.com/pkg/errors"
+	cbg "github.com/whyrusleeping/cbor-gen"
+
+	runtime "github.com/filecoin-project/specs-actors/actors/runtime"
+)
+
+// IntMap stores key-value pairs, with integer keys, in a AMT.
```

Comment:
This type will become redundant with the changes in #84 to support sparse arrays. I'll delete this and replace with Array if that merges first.